db-developments/sql/_export.sql

```diff
@@ -11,13 +11,13 @@ WHERE (Date_Complete::date <=  :'CAPTURE_DATE'
 		AND Date_Filed::date <=  :'CAPTURE_DATE'));
 
 -- EXPORT HousingDB
-DROP TABLE IF EXISTS EXPORT_housing;
-SELECT * INTO EXPORT_housing
+CREATE VIEW export_housing AS 
+SELECT * 
```

Comment:
These are very redundant tables that are pretty large, so thought it would just make sense to make them views instead and save the space. They still query quite fast